### ISSUE TITLE
English installation manual?

### ISSUE THREAD

**Issue by jamespatriot** [axelor/axelor-open-suite#12256]
Manuals in French only

**Comment by douglasg14b** [comment]
Yeah, the user docs are in french even when English is selected, which.... definitely does not lend confidence to a platform you might invest in and pay for.

https://docs.axelor.com/aos/en/sales/client_file/
